packages/core-utils/src/route.ts

```diff
@@ -304,19 +313,21 @@ function alphabeticShortNameComparator(a: Route, b: Route): number {
 export function makeNumericValueComparator(
   objGetterFn?: (item: Route) => number
 ) {
-  /* Note: Using the global version of isNaN (the Number version behaves differently. */
-  /* eslint-disable no-restricted-globals */
   return (a: number, b: number): number => {
     const { aVal, bVal } = getSortValues(objGetterFn, a, b);
-    if (typeof aVal !== "number" || typeof bVal !== "number") return 0;
```

Comment:
Instead of removing this line, should we just set the value to not be null?

Reply:
I think having `null` as a default is best because in OTP2 any numeric value should be a valid sort order, even -999. An agency should be able to use -999 with OTP2 and push routes to the top of the list if they want.

Reply:
Makes sense!

Reply:
That being said why are we using null instead of 0? Isn't 0 more precise?

Reply:
0 is not only a valid number, it's actively being used as a `sortOrder` value in some deployments!